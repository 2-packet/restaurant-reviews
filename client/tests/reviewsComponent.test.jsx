import React from 'react';
import renderer from 'react-test-renderer';
import Reviews from '../client/src/components/reviewsComponent.jsx';

it('renders correctly', () => {
  const tree = renderer
    .create(<Reviews />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
