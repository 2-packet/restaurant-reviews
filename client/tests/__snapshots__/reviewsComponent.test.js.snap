// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Reviews
  </h1>
</div>
`;
